### ISSUE TITLE
Bookmarks sidesheet loading when data is fetched

### ISSUE THREAD

**Issue by Gustav-Eikaas** [equinor/fusion-framework#863]
![image](https://user-images.githubusercontent.com/89254170/234868294-71cf87a7-1bf9-4f28-8bb3-745e558127f2.png)

This is not the only thing missing, as for now there is no loading information provided to the component but it should be fixable, but one more thing if the user have no bookmarks for the given subsystem the user should be presented witt a message  

manybe:

No bookmarks created, click on add bookmark to create your fist bookmark
updates to this has to be done in https://github.com/equinor/fusion-framework

**Comment by odinr** [comment]
@Noggling is this still a issue?

**Comment by Noggling** [comment]
@odinr yes, i hvae not looked in to this. its on my list of thigs to do.

**Comment by Gustav-Eikaas** [comment]
any update on this? The issue still persists

**Comment by atlewee** [comment]
I still get this in production, never seen my bookmarks.
A user also just reported this, that bookmarks does not work/spins forever.
